packages/davinci-client/src/lib/config.types.ts

```diff
@@ -7,7 +7,7 @@
 /**
  * Import ConfigOptions type from the JavaScript SDK
  */
-import type { AsyncConfigOptions } from '@forgerock/javascript-sdk/src/config/interfaces';
+import type { AsyncConfigOptions } from '@forgerock/shared-types';
 import { WellknownResponse } from './wellknown.types.js';
 
 export interface DaVinciConfig extends AsyncConfigOptions {
```

Comment:
Should `responseType` on line 14 be of type `ResponseType` from the authorization types?